setup.py

```diff
@@ -66,6 +67,33 @@ def read(fname):
         return f.read()
 
 
+extras_require = {
+    'admindocs': ['docutils'],
+    'argon2': ['argon2-cffi>=16.1.0'],
+    'bcrypt': ['bcrypt'],
+    'geoip': ['geoip2'],
+    'gis': ['geoip2', 'numpy'],
+    'images': ['pillow'],
```

Comment:
For Pillow it would make sense to specify version `>= 5.2.0` as the [support matrix](https://pillow.readthedocs.io/en/latest/installation.html#notes) for that version lines up with Django pretty well.

Reply:
Sounds sensible. Also, given Django 2.2 will require Python 3.5+, I've also tweaked the following:

- `numpy>=1.11.0`: v1.11.0 was the first version to support Python 3.5
- `pylibmc>=1.4.0`: v1.4.0 introduced support for Python 3
- `pyyaml>=3.08`: v3.08 was the first version to support Python 3

Do you have any thoughts on the other questions I raised above?